### ISSUE TITLE
NVDA in the cloud

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#2926]
**Reported by Q on 2013-01-11 22:07**
Allow users to create a simple NVDA-project account which would be used to store configuration, user's donation status, and other details in a central location. I am willing to work on an open source web backend for this as it would be rather simple with Flask. It would allow an NVDA user to log in with their account to their screen reader, and have all their information automatically retrieved and their screen reader configured to their preferences, including any add-ons they might have installed. It would also eliminate the minor annoyance of being prompted to donate on every update when the user is already a monthly contributor. As the backend will be open source, there are no privacy concerns, or even concerns about NV access having to maintain the infrastructure as anyone could run it on their own server should they choose.
Anticipating objections: This should be as lite as possible. I'm not requesting something deeply-integrated with NVDA, users don't even have to know about it if they don't want to. Perhaps simply a login option in the NVDA menu, no automatic prompting to log in or anything absurd on start.

**Comment by bhavyashah** [comment]
@jcsteh What is NV Access's current VPS hosting plan? In case this has improved since https://github.com/nvaccess/nvda/issues/2926#issuecomment-155303872, we may want to reconsider this ticket.

**Comment by jcsteh** [comment]
This is something we might consider in future, but our thinking is that it would be part of a low cost monthly subscription of some sort which would also include other services (yet to be determined). This would allow us to ensure we can scale with future demand and also create a small revenue stream.

**Comment by seanbudd** [comment]
Closing in favour of the more modern #16565
